tensorflow/python/ops/math_grad.py

```diff
@@ -295,8 +295,11 @@ def _LgammaGrad(op, grad):
 
 
 @ops.RegisterGradient("Digamma")
-def _DigammaGrad(op, grad):  # pylint: disable=unused-argument
-  raise NotImplementedError("grad(Digamma) == Polygamma(1) is not implemented")
+def _DigammaGrad(op, grad):
+  """Compute gradient of the digamma function with respect to its argument."""
+  x = op.inputs[0]
+  with ops.control_dependencies([grad.op]):
+    return grad * math_ops.polygamma(array_ops.ones(array_ops.shape(x), dtype=x.dtype), x)
```

Comment:
i *think* polygamma will distribute correctly if you just use constant(1, dtype=x.dtype)